common/anomalies/hfe_anomaly.txt

```diff
@@ -53,7 +54,9 @@ hfe_anomaly_criopods = {
 			from.owner = {
 				num_pops > 0
 				or = {
+					species_portrait = "human"
```

Comment:
You should make scripted trigger for it.

Reply:
No need. The flags and such are still there and unchanged. This applies to ALL men with that portrait.